Customizable Tooltip background (#737)
Phobos-developers/Phobos: docs/User-Interface.md

```{note}
Same as with harvester counter, you can download the improved font (v4 and higher; can be found on [Phobos supplementaries repo](https://github.com/Phobos-developers/PhobosSupplementaries)) or draw your own icons.
```

> - Also, the background color and opacity of tooltips can be customized by player's side.

Reviewer: ```suggestion
- The background color and opacity of tooltips can now be customized globally or per side.
```

In `rulesmd.ini`:
```ini
[SOMESIDE]
ToolTip.Background.Color=0,0,0      ; integer - R,G,B, defaults to [AudioVisual]->ToolTip.Background.Color
ToolTip.Background.Opacity=100      ; integer, ranged in [0, 100], defaults to [AudioVisual]->ToolTip.Background.Opacity

> ToolTip.Background.BlurSize=0.0     ; float, defaults to [AudioVisual]->ToolTip.Background.BlurSize

Reviewer: Defaults should be explicitly listed too.

```

```{note}

> The blur effect is resource intensive. Please make sure if you want to enable this effect. Leave it to 0.0 to disable this effect.

Reviewer: ```suggestion
The blur effect is resource intensive. Please make sure you really want to enable this effect, otherwise leave it to 0.0 so it stays disabled.
```

```

In `RA2MD.ini`:
```ini
[Phobos]

> ToolTipBlur=false ; boolean, whether the blur effect of tooltips will be enabled.

Reviewer: ```suggestion
ToolTipBlur=false  ; boolean, whether the blur effect of tooltips will be enabled.
```
